ambry-clustermap/src/main/java/com/github/ambry/clustermap/MySqlReportAggregatorTask.java

```diff
@@ -110,32 +114,40 @@ public Metrics(MetricRegistry registry) {
 
   @Override
   public TaskResult run() {
-    Pair<StatsSnapshot, StatsSnapshot> results = null;
     Exception exception = null;
     Histogram fetchTimeMs = statsReportType == StatsReportType.ACCOUNT_REPORT ? metrics.accountStatsFetchTimeMs
         : metrics.partitionClassStatsFetchTimeMs;
     Histogram aggregationTimeMs =
         statsReportType == StatsReportType.ACCOUNT_REPORT ? metrics.accountStatsAggregationTimeMs
             : metrics.partitionClassStatsAggregationTimeMs;
     long startTimeMs = System.currentTimeMillis();
+    StatsSnapshot accountPhysicalStorageSnapshot = null;
     try {
       List<String> instanceNames = manager.getClusterManagmentTool().getInstancesInCluster(manager.getClusterName());
-      Map<String, StatsWrapper> statsWrappers =
-          statsReportType == StatsReportType.ACCOUNT_REPORT ? fetchAccountStatsWrapperForInstances(instanceNames)
-              : fetchPartitionClassStatsWrapperForInstances(instanceNames);
-      fetchTimeMs.update(System.currentTimeMillis() - startTimeMs);
-      logger.info("Aggregating stats from " + statsWrappers.size() + " hosts");
-      results = clusterAggregator.doWorkOnStatsWrapperMap(statsWrappers, statsReportType);
       if (statsReportType == StatsReportType.ACCOUNT_REPORT) {
+        Map<String, HostAccountStorageStatsWrapper> accountStatsWrappers =
+            fetchAccountStorageStatsWrapperForInstances(instanceNames);
+        fetchTimeMs.update(System.currentTimeMillis() - startTimeMs);
+        logger.info("Aggregating stats from " + accountStatsWrappers.size() + " hosts");
```

Comment:
not a blocker to merge since this code was already there, but we can change this to use the formatting expression instead of string concat later.